src/isar_exr/robotinterface.py

```diff
@@ -155,21 +163,27 @@ def step_status(self) -> StepStatus:
     def stop(self) -> None:
         try:
             self.api.pause_current_mission(self.exr_robot_id)
-        except RobotException:
-            raise
-        except Exception as e:
-            raise RobotException from e
+        except Exception:
+            message: str = "Could not stop the running mission"
+            self.logger.error(message)
+            raise RobotException(
```

Comment:
Why not raise `RobotCommunicationException` and just pass the error description?